tests/scripts/build-release-downstream.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -ex
+
+# Load dot env file if available
+if [ -f .env ]; then
```

Comment:
Is this just for local dev testing? The CI doesn't use the .env, right?

Reply:
Removed